webpack failed to compile for Vue.js monaca preview

Steps to repro:

1. monaca create test
2. choose "Onsen UI and Vue.js"
3. choose any of the templates
4. cd test
5. monaca preview (or any of the commands)

Expected:

See a preview of the app.

Actual:

Webpack failed to compile, here's the output:
```
$ monaca preview
Build completed in 7.252s

chunk    {0} bundle.js, main.css (main) 4.19 MB [rendered]

ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./img/ios-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
 @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:60-102

ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./img/android-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
 @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:502-548

ERROR in ./~/onsenui/css-components-src/src/onsen-css-components.css
Module build failed: ModuleNotFoundError: Module not found: Error: Cannot resolve 'file' or 'directory' ./img/ios-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
    at C:\Users\Youdaman\.cordova\node_modules\webpack\lib\Compilation.js:229:38
    at onDoneResolving (C:\Users\Youdaman\.cordova\node_modules\webpack\lib\NormalModuleFactory.js:29:20)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\lib\NormalModuleFactory.js:85:20
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:726:13
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:52:16
    at done (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:241:17)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:44:16
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:723:17
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:167:37
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:24:19
    at onResolved (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:38:18)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:127:10
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:191:15
    at applyPluginsParallelBailResult.createInnerCallback.log (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:110:4)
    at loggingCallbackWrapper (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\createInnerCallback.js:21:19)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\tapable\lib\Tapable.js:134:6
    at Tapable.<anonymous> (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\DirectoryDefaultFilePlugin.js:21:12)
    at Storage.provide (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:52:20)
    at CachedInputFileSystem.stat (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:132:20)
    at Tapable.<anonymous> (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\DirectoryDefaultFilePlugin.js:18:6)
    at Tapable.applyPluginsParallelBailResult (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\tapable\lib\Tapable.js:139:14)
    at Tapable.<anonymous> (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:103:8)
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 541 kB [rendered]
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 405 kB [rendered]
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 386 kB [rendered]

    ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
    Module not found: Error: Cannot resolve 'file' or 'directory' ./img/ios-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
     @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:60-102

    ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
    Module not found: Error: Cannot resolve 'file' or 'directory' ./img/android-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
     @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:502-548
webpack: Failed to compile.
```

--- a/src/template/webpack.dev.angular2.js
+++ b/src/template/webpack.dev.angular2.js
@@ -70,7 +70,7 @@ module.exports = {
         path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
         path.join(__dirname, 'src')
       ],
-      loader: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
+      loaders: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
     }, {
       test: /\.css$/,
       exclude: [

--- a/src/template/webpack.dev.angular2.js
+++ b/src/template/webpack.dev.angular2.js
@@ -66,12 +66,18 @@ module.exports = {
       loader: 'file?name=assets/[name].[hash].[ext]'
     }, {
       test: /\.css$/,
-      include: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
-      loaders: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
+      include: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
+      loader: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
     }, {
       test: /\.css$/,
-      exclude: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
-      loader:  ExtractTextPlugin.extract('style', 'css?sourceMap')
+      exclude: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
+      loader: ExtractTextPlugin.extract('style', 'css?sourceMap')
     }, {
       test: /\.json$/,
       loader: 'json'

--- a/src/template/webpack.dev.react.js
+++ b/src/template/webpack.dev.react.js
@@ -79,11 +79,17 @@ module.exports = {
       loader: 'file?name=assets/[name].[hash].[ext]'
     }, {
       test: /\.css$/,
-      include: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      include: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')
     }, {
       test: /\.css$/,
-      exclude: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      exclude: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?sourceMap')
     }, {
       test: /\.json$/,

--- a/src/template/webpack.dev.vue.js
+++ b/src/template/webpack.dev.vue.js
@@ -67,11 +67,17 @@ module.exports = {
       loader: 'file?name=assets/[name].[hash].[ext]'
     }, {
       test: /\.css$/,
-      include: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      include: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')
     }, {
       test: /\.css$/,
-      exclude: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      exclude: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?sourceMap')
     }, {
       test: /\.json$/,

--- a/src/template/webpack.prod.angular2.js
+++ b/src/template/webpack.prod.angular2.js
@@ -71,12 +71,18 @@ module.exports = {
       loader: 'file?name=assets/[name].[hash].[ext]'
     }, {
       test: /\.css$/,
-      include: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
-      loaders: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
+      include: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
+      loader: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
     }, {
       test: /\.css$/,
-      exclude: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
-      loader:  ExtractTextPlugin.extract('style', 'css?sourceMap')
+      exclude: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
+      loader: ExtractTextPlugin.extract('style', 'css?sourceMap')
     }, {
       test: /\.json$/,
       loader: 'json'

--- a/src/template/webpack.prod.angular2.js
+++ b/src/template/webpack.prod.angular2.js
@@ -75,7 +75,7 @@ module.exports = {
         path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
         path.join(__dirname, 'src')
       ],
-      loader: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
+      loaders: ['css-to-string', ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')]
     }, {
       test: /\.css$/,
       exclude: [

--- a/src/template/webpack.prod.react.js
+++ b/src/template/webpack.prod.react.js
@@ -81,11 +81,17 @@ module.exports = {
       loader: 'file?name=assets/[name].[hash].[ext]'
     }, {
       test: /\.css$/,
-      include: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      include: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')
     }, {
       test: /\.css$/,
-      exclude: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      exclude: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?sourceMap')
     }, {
       test: /\.json$/,

--- a/src/template/webpack.prod.vue.js
+++ b/src/template/webpack.prod.vue.js
@@ -69,11 +69,17 @@ module.exports = {
       loader: 'file?name=assets/[name].[hash].[ext]'
     }, {
       test: /\.css$/,
-      include: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      include: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?importLoaders=1&-raw!postcss')
     }, {
       test: /\.css$/,
-      exclude: [/\/onsen-css-components.css$/, path.join(__dirname, 'src')],
+      exclude: [
+        path.join(__dirname, 'node_modules', 'onsenui', 'css-components-src', 'src'),
+        path.join(__dirname, 'src')
+      ],
       loader: ExtractTextPlugin.extract('style', 'css?sourceMap')
     }, {
       test: /\.json$/,

